data/hai/hai reveal 5 philosophers.txt

```diff
@@ -0,0 +1,1075 @@
+			choice
+				`	Follow them.`
+			`	Scanners show the room to be a vast cavern; some sort of meeting room suitable for ten thousand people or more. It is pitch black when you come in. Once all operatives are inside, a second blast door closes behind you and lights turn on. The room is as bright as day.`
+			`	You see before you several hundred pirate corpses a few thousand heavily-armed civilians all pointing guns at the operatives. It seems the citizens have arranged a violent rebellion to coincide with the Republic's attack. Three unarmed civilians walk towards the operatives: one Hai, one human, and one Korath. The leader of the Republic operatives exits her suit and approaches them.`
+			`	The Hai speaks, "We are the leaders of the civilian government of Philosopher's Paradise. Although we are grateful for your assistance in defeating our violent rulers, we will not accept your dominion either. We know what you did to the Free Worlds. Philosopher's Paradise is an independent nation; we will never join the Republic."`
+			`	"I promise our country will turn to peace, and we will send no military ships through the wormhole."`
+			`	After several hours of discussions, the operatives leave their suits and surrender, preparing to return peacefully to Republic space.`
+			`	Do you want to talk to the Philosopher's Paradise citizens before you go?`
+			choice
+				`	(Talk to them.)`
+				`	(Don't talk to them.)`
+					goto leave
+			`	You approach what looks like a group of leaders: one human, one Hai, and one Korath. The human turns to you and says, "You look like an asker of questions. How may we enlighten you?"`
+				goto "actual questions"
+			label questions
+			`	"Do you have more questions?"`
+			label "actual questions"
+			choice
+				`	"Do you want me to take you home?"`
+					goto home
+				`	"Do you know anything about the microtranceivers?"`
+					goto "microtranceivers"
+				`	"What can you tell me about this city?"`
+					goto place
+				`	"What do you do here?"`
+					goto do
+				`	"What do you know about this area of space?"`
+					goto space
+				`	"How did the Korath get here?"`
+					goto korath
+				`	"Can you speak to other Korath for me?"`
+					goto "translation services"
+				`	"Why were there Korath Raiders in this system?"`
+					goto korath
+				`	"I have no more questions."`
+			`	"It was a pleasure meeting you," says the human leader.`
+				goto leave
+			label korath
+			`	The Korath leader tells you, "A few years ago, three of our ships jumped into the Alcyone system to raid. Numerous pirate vessels left Stormhold and disabled all of our ships. A large human ship boarded our flagship. All we heard was screaming; the pirates used deadly gas to kill the entire crew."`
```

Comment:
Hmm. Are Korath even able to phonetically pronounce human languages? Ya know, I have a whole mission chain about a Korath translation device that's currently shut down (maybe not _because_ it's a Korath translation device though).